specs/commandline/inspect.md

```diff
@@ -0,0 +1,234 @@
+
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with the signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+└──application/vnd.cncf.notary.signature
+  ├──<digest_of_signature_manifest>
+    ├──<signing algorithm>
+    ├──<signed attributes>
+    ├──<user-defined attributes>
+    ├──<unsigned attributes>
+    ├──<certificates>
+    └──<payload>
+  ├──<digest_of_signature_manifest>
+    ├──<signing algorithm>
+    ├──<signed attributes>
+    ├──<unsigned attributes>
+    ├──<certificates>
+    └──<payload>
+```
+
+## Outline
+
+```text
+Inspect all signatures with the signed artifact.
+
+Usage:
+    notation inspect [flags] <reference>
+  
+Flags:
+   -h, --help for describing the signature
+   -o, --output on command line sets the output to json
```

Comment:
Since we need to add this kind of `output` flag for other notation cli commands, and it is on the user interface, it's better we can align what is the best solution for our users. Here are 3 options so far:

1. `-o, --output json`
2. `--display {tree, json) by default tree
3. --json (an example in https://clig.dev/#output)

Any other options, Let's vote?